templates/common/infra/bicep/core/database/mysql/mysql-database.bicep

```diff
@@ -0,0 +1,18 @@
+param name string = ''
```

Comment:
please use mysql-db.bicep file name

please have a look at how I'm doing cosmos-mongo-db.  I create the account there so the end user only needs to call cosmos-mongo-db and doesn't have to create an account as well.

Reply:
thanks, I tried this way in commit https://github.com/wangmingliang-ms/spring-petclinic-java-mysql/commit/8f22d235368949ae518a4c2056fc224562d6a63d, but it keeps `Compiling Bicep template...` when I run `azd provision` locally, do you know what is wrong?

Reply:
Are you still seeing this issue?

Reply:
I fixed it. This issue happened because the param name in `main.paramters.json` and `main.bicep` is not consistent. 
But there are no messages telling me what is wrong, it just keep `Compiling Bicep template...`. 
is it possible for you to improve the experience? so that the user knows how to fix it